src/Data/Characters/YunJin/index.tsx

```diff
@@ -172,7 +172,10 @@ const sheet: ICharacterSheet = {
         node: infoMut(dmgFormulas.skill.shield, { key: `char_${key}_gen:skill.skillParams.3` }),
       }, {
         text: tr("skill.skillParams.4"),
-        value: data => data.get(input.constellation).value < 1 ? `${datamine.skill.cd}s` : `${datamine.skill.cd}s - 18%`,
+        value: data => data.get(input.constellation).value >= 1
```

Comment:
I wonder if its easier for the end user to read `9s - 18%` or `7.38s` without showing how that is calculated...

Reply:
```suggestion
        value: data => data.get(input.constellation).value >= 1
          ? `${datamine.skill.cd} - 18% = ${datamine.skill.cd*(1-0.18)}`
          : `${datamine.skill.cd}`,
```
In screenshot below, (1) is current implementation.
(2) in screenshot: We could use a WR node to represent the CD, in which case the '?' will appear with the formula. We would need a workaround to add the "s" unit for a WR node display though.
(3) in screenshot: We could show the equation as part of the text
![image](https://user-images.githubusercontent.com/36019388/154860378-16f88c98-f767-4426-a5bf-61182443a0f4.png)

Bennett's formula for A1 and A4-in-ult will be:
`cd*(1 + -1*0.8*0.5)`, or `cd - 20% - 50%`, which could also be represented as `cd - 60%` (I checked in game, the cooldown factors get multiplied, not added).

Not sure if those calculations will be confusing for end users since it doesn't logically add up how 20% and 50% CD reduction will create 60% unless we represent them as CD factors (aka `1-(80%*50%)`), but maybe that will also be confusing

Reply:
I'd say 3rd option for now. I dont really like the idea to boggling WR with these weird calculations that doesn't contribute much

Reply:
Changed it to 3rd option for Bennett, Yunjin and Kazuha